lib/create-testing-library-rule/detect-testing-library-utils.ts

```diff
@@ -606,9 +613,11 @@ export function detectTestingLibraryUtils<
         isTestingLibraryUtil(
           identifierNode,
           (identifierNodeName, originalNodeName) => {
-            return [identifierNodeName, originalNodeName]
-              .filter(Boolean)
-              .includes('debug');
+            return (
+              (validNames as string[]).includes(identifierNodeName) ||
+              (!!originalNodeName &&
+                (validNames as string[]).includes(originalNodeName))
+            );
```

Comment:
```suggestion
            return [identifierNodeName, originalNodeName]
              .filter((nodeName): nodeName is string => Boolean(nodeName))
              .some((nodeName) => (validNames as string[]).includes(nodeName));
```

Reply:
I'm not sure if I'd call that simpler; but I'd prefer to keep it how it is currently because it matches how the same conditional is laid out in other utils (i.e `isAsyncUtil`)

Reply:
Let's see what @Belco90 says, but I think we should update `isAsyncUtil` too (in a separate PR) as this is much cleaner an easier to understand imo

Reply:
Totally! in fact, you could even deduplicate the code with a little local function that accepts the node + an array of strings as names - something like this:

```
    const isTestingLibraryUtilOfName = (
      node: TSESTree.Identifier,
      validNames: string[]
    ): boolean => {
      return isTestingLibraryUtil(
        node,
        (identifierNodeName, originalNodeName) => {
          return (
            validNames.includes(identifierNodeName) ||
            (!!originalNodeName && validNames.includes(originalNodeName))
          );
        }
      );
    };
```

Reply:
Extracting it in that new PR sounds indeed a better idea too